### ISSUE TITLE
Update Project Profile: Lucky Parking (Remove Yen Phan)

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#5807]
### Prerequisites
1. Be a member of Hack for LA. (There are no fees to join.) If you have not joined yet, please follow the steps on our [Getting Started page](https://www.hackforla.org/getting-started) and attend an onboarding session.
2. Before you claim or start working on an issue, please make sure you have read our [How to Contribute to Hack for LA Guide](https://github.com/hackforla/website/blob/gh-pages/CONTRIBUTING.md).

### Overview
We need to keep project information up to date so that visitors to the website can find accurate information.

### Action Items
- [x] In your IDE, open the `_projects/lucky-parking.md` file
- [x] Observe the existing syntax of the front matter block [^1] in the file
- [x] Find the `leadership` variable and remove the following:
```
  - name: Yen Phan
    role: User Research
    links:
      slack: 'https://hackforla.slack.com/team/U025M9VRDJR'
      github: 'https://github.com/ymphan'
    picture: https://avatars.githubusercontent.com/ymphan
```
- [x] Verify the changes by viewing the following in your local environment and include before and after screenshots with your pull request:
  - [x] Lucky Parking page [^2]
- [x] Once your pull request is merged, go to #4909 and
  - [x] Check off this issue under the _Dependency_ section
  - [ ] If all the issues in the _Dependency_ section are checked off, move #4909 to **Questions / In Review** column and uncheck the **Dependency label**.
    - <details>
        <summary>Click here to see how to uncheck the <b>Dependency label</b></summary>
        <img src="https://github.com/hackforla/website/assets/31293603/6f53f4d4-7d2c-45f8-8534-9936fc9adee8" width="300px">
      </details>

### Resources/Instructions
- This issue was initiated by #4909
[^1]: [Info about the front matter block](https://jekyllrb.com/docs/front-matter/)
[^2]: Project detailed info page URL: https://www.hackforla.org/projects/lucky-parking

**Comment by james-aguirre** [comment]
Hello, 
- I'm working on this issue at the moment.
- PR should be in by tonight.

**Comment by roslynwythe** [comment]
Hi @james-aguirre - developers can only work on one issue at a time, so I need to unassign you from this issue.  When your existing pull request is merged, you may take on another issue.  Thank you

**Comment by junadkat32** [comment]
Availability: Tues, Wed, Thursday: 9-5 pm EST.
ETA: 18th Nov, 2023

**Comment by ExperimentsInHonesty** [comment]
@junadkat32 I moved this to the in progress column.  You can do that yourself in the future when you self assign.  Cheers!
